wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/ArrayOfXRefMessageWrapper.java

```diff
@@ -0,0 +1,73 @@
+package org.hpccsystems.ws.client.wrappers;
+
+import java.io.ByteArrayInputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+
+public class ArrayOfXRefMessageWrapper
+{
+    protected List<XRefMessageWrapper> messages = null;
+    protected String cluster = null;
+
+    public String getCluster()
+    {
+        return cluster;
+    }
+
+    public ArrayOfXRefMessageWrapper (String response)
+    {
+        messages = new ArrayList<XRefMessageWrapper>();
+
+        if (response != null)
+        {
+            try
+            {
+                DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+                DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+
+                Document doc = dBuilder.parse(new ByteArrayInputStream(response.getBytes("UTF-8")));
+
+                NodeList xrefnodelist = doc.getElementsByTagName("Warning"); //there could be others, info?
```

Comment:
Same question about constants. Any reason not to add all child elements here instead of looking for specific tags?